src/ripple/basics/ScopedLock.h

```diff
@@ -51,7 +54,7 @@ namespace ripple
         {
             ... do some stuff with it locked ..
 
-            GenericScopedUnlock<std::mutex> unlocker{mut};
+            ScopedUnlock<std::mutex> unlocker{myScopedLock};
```

Comment:
Given class template argument deduction, you should probably remove the `<std::mutex>` from the example.